app/src/main/java/com/example/criminalintent/presentation/crimeDetail/CrimeFragment.kt

```diff
@@ -109,7 +112,7 @@ class CrimeFragment : Fragment(), FragmentResultListener {
             R.id.add_crime -> {
                 viewModel.addOrUpdateCrime()
                 viewModel.addOrUpdatePhotoFile()
-                activity?.supportFragmentManager?.popBackStack()
+                navController.popBackStack()
```

Comment:
I think we should navigate exactly where we want to in this case as well and not rely on the back stack